frontend/src/common/queries/entities.ts

```diff
@@ -9,4 +9,14 @@ export enum ENTITIES {
   WMG_FILTERS_QUERY = "WMG_FILTERS_QUERY",
   WMG_MARKER_GENES = "WMG_MARKER_GENES",
   WMG_GENE_INFO = "WMG_GENE_INFO",
+  CELL_EXPLORER_CELL_CARDS = "CELL_EXPLORER_CELL_CARDS",
```

Comment:
@ashin-czi 
Let's rename `CELL_EXPLORER` to `CELL_CARDS`. I called it CELL_EXPLORER when I thought we were naming it that, but let's use CELL_CARDS to be consistent.